tools/make-index-md.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+
+set -eu
+
+cd "$@"/docs/
+(
+        echo "# systemd Documentation"
+
+        for f in *.md ; do
+                if [ "x$f" != "xindex.md" ] ; then
+                        t=`grep "^# " "$f" | head -n 1 | sed -e 's/^#\s*//'`
+                        echo -e "\n* [$t]($f)"
```

Comment:
Pffff, this is only necessary because markdown generally does not support extracting the title from the link target. Sphinx does this for internal links without any fuss, and imho this saves a lot of work. But anyway, this is not relevant here.